sig-architecture/charter.md

```diff
@@ -0,0 +1,43 @@
+# SIG Architecture Charter
+
+## Scope
+
+The Architecture SIG maintains and evolves the design principles of KubeVirt,
+and provides a consistent body of expertise necessary to ensure architectural
+consistency over time.
+
+### In scope
+
+#### Code, Binaries, Docs, and Services
```

Comment:
Will "Quality prolicy" fit here as well? E.g. code quality and testing policies.

I would also consider SIG management, e.g. considering to create, modify and remove SIGs and/or WGs.